build_deploy.sh

```diff
@@ -0,0 +1,32 @@
+set -exv
+
+IMAGE="quay.io/cloudservices/content-sources-backend"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+SMOKE_TEST_TAG="latest"
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
+    echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN  must be set"
+    exit 1
+fi
+
+export DOCKER_LOGIN_USER=$QUAY_USER
+export DOCKER_LOGIN_TOKEN=$QUAY_TOKEN
+export DOCKER_REGISTRY="quay.io"
+make docker-login
+
+export DOCKER_LOGIN_USER=$RH_REGISTRY_USER
+export DOCKER_LOGIN_TOKEN=$RH_REGISTRY_TOKEN
+export DOCKER_REGISTRY="registry.redhat.io"
+make docker-login
```

Comment:
```suggestion
make docker-login \
    DOCKER_LOGIN_USER="$RH_REGISTRY_USER" \
    DOCKER_LOGIN_TOKEN="$RH_REGISTRY_TOKEN" \
    DOCKER_REGISTRY="registry.redhat.io"
```
Similar to the above one.